ocs_ci/ocs/utils.py

```diff
@@ -733,8 +733,12 @@ def run_must_gather(log_dir_path, image, command=None):
         image (str): must-gather image registry path
         command (str): optional command to execute within the must-gather image
     """
+    # Must-gather has many changes in 4.6 which add more time to the collection.
+    # https://github.com/red-hat-storage/ocs-ci/issues/3240
+    ocs_version = float(ocsci_config.ENV_DATA['ocs_version'])
+    timeout = 1500 if ocs_version == 4.6 else 600
```

Comment:
I suspect this will still be the case for 4.7 and beyond, let's ensure we're keeping this logic going forward.

```suggestion
    timeout = 1500 if ocs_version >= 4.6 else 600
```